### DISCUSSION TITLE
Client won't start

### DISCUSSION THREAD

**Discussion by Goldi82PL** [fedorovvl/tso_client#261] [Q&A]
I need help, since yesterday I can't start the client 🙁  all the time I only have start page with full loaded  bar. I redownloaded, clear data, but it didn't help. Pls help

**Comment by Maetel2002** [comment]
Exactly the same for me on my FR and my European English accounts. 

Works at 10.00Am Fr time and since 11.30AM, the client remain on the mid-bar charge and nothing more... (See attachement)

Unity log on is working... But...

![Capture d’écran 2023-04-30 124727](https://user-images.githubusercontent.com/131248724/235348935-99b7c17c-45e3-4a48-84fe-66f02dccd09e.jpg)

**Reply by skelgaard** [reply]
have you tested in unity that it works there ?

**Reply by Maetel2002** [reply]
As I wrote, I have no problem to log on on Unity with my both accounts and it works... like Unity is able to work...

**Reply by Cdrwbw** [reply]
![image](https://user-images.githubusercontent.com/111175316/235353769-d07b4aa6-bd8d-4ece-b89c-1df8b2c4170d.png)
Sometimes when closing the client, a process does not close right away and when restarting the client it can block.
Check if this process is not actually running.
Close it with end of task if necessary.

**Comment by Maetel2002** [comment]
No running process...

**Comment by Maetel2002** [comment]
Oversized Windows update which used all the bandwidth in the background !! Now, Client is back to work for me !!

**Reply by Cdrwbw** [reply]
Yes, sometimes it's annoying, and I've reduced the bandwidth for updates to avoid this kind of inconvenience

**Comment by Maetel2002** [comment]
Got it in my list now... Will check it on next time issue !!! 
Tks all for your support ;)

**Comment by Goldi82PL** [comment]
Still dont work... Once again clear all data (tso folder and client) install one more time but nothing changes
![Przechwytywanie](https://user-images.githubusercontent.com/132189973/235370432-da8492d6-5006-459b-b711-1eacf4e80add.JPG)

**Reply by Cdrwbw** [reply]
It's lasted version of client ?

**Reply by Goldi82PL** [reply]
yes lasted one 1.5.6.2

**Reply by fedorovvl** [reply]
99% network problem.. vpn? proxy?

**Reply by Goldi82PL** [reply]
I haven't changed anything and I use the same network access all the time :/ 2 days ago everythink works fine but from yesterday i have this issue 

Edit // just star working :O

**Comment by Anprivate** [comment]
I have the connection problem since yesturday, I suppose something was blocked by our local regulator (РКН), so client doesn't start properly. With VPN the client works ok. But I don't like to use VPN - I prefer proxy. Is there any way to run client with proxy?

**Reply by fedorovvl** [reply]
try to set system proxy

**Reply by Anprivate** [reply]
Yes, it is working, but I don't want to send all the traffic through the proxy. I prefer to send to proxy only trafffic specified by me
